e2e/api/api_test.go

```diff
@@ -45,6 +38,9 @@ var _ = Describe("server API validation", func() {
 	)
 
 	BeforeEach(func() {
+		StartEtcd()
+		StartMerlin()
```

Comment:
Moved these into BeforeEach because the NotFound/AlreadyExists relies on state in etcd. So we need to flush it out after every test.